ArduCopter/Parameters.cpp

```diff
@@ -1505,3 +1516,32 @@ void Copter::convert_tradheli_parameters(void)
 
 }
 #endif
+
+void Copter::convert_fs_options_params(void)
+{
+    // If FS_OPTIONS is not zero, then it has already been configured and we don't change it.
+    if (g2.fs_options != 0) {
+        return;
+    }
+
+    // Variable to capture the new FS_OPTIONS setting
+    AP_Int32 fs_options_converted;
```

Comment:
we shouldn't be declaring an AP_xxx parameter on the stack, just use a int32_t

Reply:
Done